scripts/generator/graphql_generator/csharp/SDK/Android/AndroidNativeCheckout.cs.erb

```diff
@@ -23,7 +24,7 @@ namespace <%= namespace %>.SDK.Android {
             CartState = cartState;
         }
 
-        public bool CanShowPaymentSetup() {
+        public bool CanShowPaymentSetup(PaymentSettings paymentSettings) {
             // TODO
             return true;
```

Comment:
Is this selecting your payment method? If so that's the wallet chooser that we show when we request the masked wallet.